base/parse-type.jl

```diff
@@ -0,0 +1,166 @@
+# Parses a string into a Julia type object, e.g. `Int`, `Array{Int, 2}`, etc.
+function Base.parse(::Type{T}, str::AbstractString) where T<:Type
+    ast = try
+        Meta.parse(str)
```

Comment:
Instead of a `try … catch` block, couldn't you pass `raise=false` to `Meta.parse`?

Reply:
Oh perfect, thank you! I was looking for `Meta.tryparse` but didn't find it, and i didn't check the parameters. 👍

Reply:
fixed, thanks